src/Microsoft.AspNetCore.Testing/xunit/FrameworkSkipConditionAttribute.cs

```diff
@@ -33,24 +33,23 @@ private static bool CanRunOnThisFramework(RuntimeFrameworks excludedFrameworks)
                 return true;
             }
 
+#if NET451
             if (excludedFrameworks.HasFlag(RuntimeFrameworks.Mono) &&
                 TestPlatformHelper.IsMono)
             {
                 return false;
             }
 
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CLR", StringComparison.OrdinalIgnoreCase))
+            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR))
             {
                 return false;
             }
-
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CoreCLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CoreCLR", StringComparison.OrdinalIgnoreCase))
+#else
```

Comment:
Yes, we should have been using [`System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription`](http://dotnet.github.io/api/System.Runtime.InteropServices.RuntimeInformation.html)